examples/tinytest/src/main.rs

```diff
@@ -124,14 +113,12 @@ fn main()
 
     let color_title                      = CString::new("Color Choose").unwrap();
     let color_default_hex                = CString::new("Color Choose").unwrap();
-    let color_default_RGB : [c_uchar ; 3] = [250,0,0];
-    let color_result_RGB  : [c_uchar ; 3] = [100,0,0];
+    let color_default_rgb                = CString::new("FFF").unwrap();
```

Comment:
This doesn't seem to match the documentation at https://github.com/jdm/tinyfiledialogs/blob/master/libtinyfiledialogs/tinyfiledialogs.h#L128 . Why this change?